### DISCUSSION TITLE
Pyarmor Error

### DISCUSSION THREAD

**Discussion by PROGRAMADORRPL** [dashingsoft/pyarmor#1953] [General]
I recently upgraded my Intel-based MacBook Air to a MacBook Air with an M3 chip and Pyarmor keeps telling me that it is not supported. Is there a way to fix this?

Are you sure to continue? (yes/no) y
INFO     process activation file "pyarmor-regcode-xxxx.txt"
INFO     send request to server
ERROR    HTTP Error 400: Bad Request
ERROR    please enable debug option `-d` to run it again

INFO     Python 3.13.0
INFO     Pyarmor 8.5.12 (trial), 000000, non-profits
INFO     Platform darwin.aarch64
INFO     search inputs ...
INFO     find script Prueba_inicio.py
INFO     find 1 top resources
ERROR    "d.a.r.w.i.n...a.a.r.c.h.6.4" is still not supported by Pyarmor
ERROR    please enable debug option `-d` to run it again

**Comment by jondy** [comment]
First make sure this license isn't Pyarmor old license for Pyarmor 7
Then take check list
https://pyarmor.readthedocs.io/en/latest/reference/solutions.html

**Reply by PROGRAMADORRPL** [reply]
Please, that doesn't solve my problem. I'm very disappointed with the license purchase. It's the most wasted 60 euros of my life.

I'm using and can't register the license:

pyarmor reg -p "Product name" pyarmor-regcode-xxxx.txt

INFO     Python 3.13.0
INFO     Pyarmor 8.5.12 (trial), 000000, non-profits
INFO     Platform darwin.aarch64
INFO     last register was within 30 seconds, waiting for 5 seconds
INFO     waiting ...
INFO     waiting ...
INFO     prepare "pyarmor-regcode-xxxx.txt"
INFO     query key file from server
This license registration information will be

License Type    : pyarmor-basic
License To      : My name
License Product : Product name

This license is about to be used for product "Product name"

IMPORTANT: PRODUCT NAME CAN NOT BE CHANGED AFTER INITIAL REGISTRATION

Are you sure to continue? (yes/no) y
INFO     process activation file "pyarmor-regcode-xxxx.txt"
INFO     send request to server
ERROR    HTTP Error 400: Bad Request
ERROR    please enable debug option `-d` to run it again


INFO     Python 3.13.0
INFO     Pyarmor 8.5.12 (trial), 000000, non-profits
INFO     Platform darwin.aarch64
INFO     search inputs ...
INFO     find script scrit.py
INFO     find 1 top resources
ERROR    "d.a.r.w.i.n...a.a.r.c.h.6.4" is still not supported by Pyarmor
ERROR    please enable debug option `-d` to run it again

**Comment by jondy** [comment]
I can't help you if you don't provide more informations.
First of all, in the check list, does bootstrap check passed?
And if all check list is past, it should provide your license no.
I can see nothing more information.

**Comment by jondy** [comment]
Anyway I found the problem.
Parmor still not supports Python3.13 now
